CONTRIBUTING.md

```diff
@@ -8,30 +8,30 @@ This is the repository for CloudQuery SDK. If you are looking for CloudQuery CLI
 
 - [Homepage](https://cloudquery.io)
 - [Documentation](https://docs.cloudquery.io)
-- [CLI and Plugins Mono Repo](https://github.com/cloudquery/cloudquery)
+- [CLI and Plugins Monorepo](https://github.com/cloudquery/cloudquery)
 - [Discord](https://cloudquery.io/discord)
 
 ## Development
 
 ### gRPC
 
-CloudQuery has a pluggable architecture and uses [gRPC](https://grpc.io/) to communicate between source plugins, CLI and destination plugins. To develop a new plugin for CloudQuery, you don’t need to understand the inner workings of gRPC as those are abstracted via the [plugin-sdk](#cloudquery-plugin-sdk-repository).
+CloudQuery has a pluggable architecture and uses [gRPC](https://grpc.io/) to communicate between source plugins, CLI and destination plugins. To develop a new plugin for CloudQuery, you don’t need to understand the inner workings of gRPC as those are abstracted via the [plugin-sdk](#packages).
```

Comment:
This internal link was broken, I changed it to `#packages` as that makes the most sense)